qutebrowser/browser/commands.py

```diff
@@ -1513,11 +1513,14 @@ def download(self, url=None, *, mhtml_=False, dest=None):
             )
 
     @cmdutils.register(instance='command-dispatcher', scope='window')
-    def view_source(self, edit=False):
+    def view_source(self, edit=False, pygments=False):
         """Show the source of the current page in a new tab.
 
         Args:
             edit: Edit the source in the editor instead of opening a tab.
+            pygments: Use pygments to generate the view. This is always the
+            case for webkit. For webengine it may display slightly different
+            source (some javascript processing applied).
```

Comment:
You'll need to indent this for the argument description parsing to work correctly. Like this:

```
            pygments: Use pygments to generate the view. This is always the
                      case for webkit. For webengine it may display slightly
                      different source (some javascript processing applied).
```

Also: webkit -> QtWebKit, webengine -> QtWebEngine, javascript -> JavaScript :wink: